travis_conf.json

```
@@ -0,0 +1,9 @@
+{
+  "db_name": "database",
+  "user": "postgres",
+  "password": "",
+  "host": "127.0.0.1",
+  "port": "5432",
+  "email": "judical.analyst@gmail.com",
+  "email_password": "axcaxs123QWE#@!"
+}
```

Новая строка в конце файла.

```suggestion
}

```